modules/core/src/v2/wallet.ts

```diff
@@ -1055,7 +1055,7 @@ export class Wallet {
       params = params || {};
       common.validateParams(params, ['address'], ['walletPassphrase', 'xprv', 'otp'], callback);
 
-      if (['eth', 'xrp'].includes(self.baseCoin.getFamily())) {
+      if (['eth', 'xrp', 'avaxc'].includes(self.baseCoin.getFamily())) {
```

Comment:
This should be the path for all non-utxo coins
```suggestion
      if (!self.baseCoin instanceof AbstractUtxoCoin) {
        // This path is for account-based coins
```